Classes/Console/Command/SchedulerCommandController.php

```diff
@@ -117,4 +117,42 @@ protected function executeSingleTask($taskId, $forceExecution)
             }
         }
     }
+
+    /**
+     * Enable a scheduler task
+     *
+     * <b>Example:</b> <code>typo3cms scheduler:enable 42</code>
+     *
+     * @param int $taskId Uid of the task that should be enabled
+     */
+    public function enableTaskCommand($taskId)
+    {
+        $task = $this->scheduler->fetchTask($taskId);
+        if ($this->scheduler->isValidTaskObject($task) && $task->isDisabled()) {
+            try {
+                $task->setDisabled(false);
+                $this->scheduler->saveTask($task);
+            } catch (\Exception $e) {
```

Comment:
You should output this message on `stderr` and exit with a failure coe. Commands that silently fail are a source of error.